library/src/helpers/bemClasses.ts

```diff
@@ -14,6 +28,30 @@ class BEMCLasses {
   concatenate(classes: string[]): string {
     return classes.filter(Boolean).join(' ');
   }
+
+  identifier(identifiers: Array<Identifier | string>): string {
+    const id = identifiers
+      .map(i => {
```

Comment:
`identifiers.map`, but identifiers can be string, is this intentional?

Reply:
`identifiers` can be Identifier or string and in map I returned always string with :)